.travis.yml

```diff
@@ -62,6 +62,23 @@ matrix:
       env: WP_VERSION=master WP_MULTISITE=0
   allow_failures:
     - php: "nightly"
+  allow_failures:
```

Comment:
Could we, perhaps, merge the two "allow_failures" together? Just another block starts two lines above. Is there a reason for not merging those?

Reply:
Of course. I was that php nightly was on allowing failure but I forgot to combine it with the new tests.